tests/pipe/test_pipe.cpp

```diff
@@ -299,22 +299,18 @@ SEQAN_DEFINE_TEST(test_pipe_test_sorter) {
     testSorter(MAX_SIZE);
 }
 
-template <typename TStringSet>
-inline void appendValues(TStringSet &stringSet, int numArgs, ...)
+template <typename TStringSet, typename T>
+inline void appendValues(TStringSet &stringSet, T t)
 {
-    va_list ap;
-    va_start(ap, numArgs);
-    for(int j = 0; j < numArgs; ++j)
-        appendValue(stringSet, va_arg(ap, char const *));
-    va_end(ap);
+    appendValue(stringSet, t);
 }
```

Comment:
This works, if you want to change it, please provide the source code. I see no benefit for a initializer_list. The change was based on http://stackoverflow.com/questions/16987493/getting-the-type-of-parameters-passed-in-a-variable-arguments-list